packages/app-store/sendgridothercalendar/README.mdx

```diff
@@ -0,0 +1,10 @@
+---
+description: SendGrid delivers your transactional and marketing emails through the world's largest cloud-based email delivery platform.
+items:
+  - /api/app-store/sendgridothercalendar/1.png
+---
+
+<Slider items={items} />
```

Comment:
Since this week I cleaned up all v1 appstore related things, which also means this `Slider` component doesn't need to be rendered anymore. I guess you started this before the cleanup was merged, because it should also be gone from the template. Could you remove it here place? 😁

Reply:
That came from the CLI if I'm not wrong cc: @hariombalhara

Reply:
Also @hariombalhara, now with the absence of the Install Button component, for apps like this that require a previous step before installing it (setup page to get an API key) when the Install button of the app gets clicked, a toast message wrongly appears stating the app was successfully installed when it stills needs to redirect first.

Reply:
I think the slider think isn't there in CLI. Not sure how it came. But I expect it to not come for an app created using CLI.

@leog what should happen according to you. According to me, the app is installed, now the configuration is pending. Because at this moment, even if you skip the setup page configuration, it would show up as installed everywhere.